jekyll-assets/_layouts/docs.html

```diff
@@ -1,25 +1,181 @@
+                              class="bold toc-item"
+                              {% else %}
+                              class="toc-item" 
+                              {% endif %}
+                            >
+                            {{ item.title | markdownify }}
+                            </a>
+                          </a>
+                        </div>
+                      </label>
+                    </span>
+                  </li>
+                  {% if item.sections[0] %}
+                  {% for entry in item.sections %}
+                  <div class="itemcontents">
+                    <ul class="sectlevel1">
+                      <input class="toc-toggle-box" type="checkbox" id="{{ item.path }}#{{ entry.anchor }}"
+                        {% if endcapture contains entry.anchor %}
+                        checked=true
+                        {% endif %}
+                      />
+                      <li>
+                        <span>
+                          <label class="toc-toggle"
+                            {% if entry.subsections[0] %}
+                            for="{{ item.path }}#{{ entry.anchor }}"
+                            {% else %}
+                            {% endif %}>
+                            <div class="toc-toggle-container">
+                              {% if entry.subsections[0] %}
+                              <a class="level3" href="{{ site.baseurl }}{{ item.path }}#{{ entry.anchor }}" >
+                                <a id="{{ item.path }}#{{entry.anchor}}-toc" class="toc-item" alt="expand documentation category" title="expand documentation category"
+                                >
+                                <style onload="if (window.location.hash.includes('{{ entry.anchor }}')) { document.getElementById('{{ item.path }}#{{entry.anchor}}-toc').setAttribute('style', 'font-weight:bold'); document.getElementById('{{ item.path }}#{{ entry.anchor }}').checked = true; }"></style>
+                                  {{ entry.heading | markdownify }}
+                                </a>
+                              </a>
+                              {% else %}
+                              <a class="level3">
+                                <a id="{{ item.path }}#{{entry.anchor}}-toc" alt="expand documentation category" title="expand documentation category"
+                                  href="{{ site.baseurl }}{{ item.path }}#{{ entry.anchor }}"
+                                  onclick="document.getElementById('mobile-toggle').checked = false;"
+                                  class="toc-item no-dropdown"
+                                  >
+                                  <style onload="if (window.location.hash.includes('{{ entry.anchor }}')) { document.getElementById('{{ item.path }}#{{entry.anchor}}-toc').setAttribute('style', 'font-weight:bold'); document.getElementById('{{ item.path }}#{{ entry.anchor }}').checked = true; }"> </style>
```

Comment:
As above re using an empty `style` tag purely to execute JavaScript.

Reply:
as above, so below!